packages/perseus/src/widgets/__tests__/passage_test.jsx

```diff
@@ -130,4 +167,48 @@ describe("passage widget", () => {
         // Assert
         expect(reference).toBe(null);
     });
+
+    it("should render passage title", () => {
+        renderPassage({passageTitle: "Hello World"});
+
+        expect(screen.getByText("Hello World")).toBeInTheDocument();
+    });
+
+    it("should render passage text", () => {
+        renderPassage({passageText: "Hello World"});
+
+        expect(screen.getByText("Hello World")).toBeInTheDocument();
+    });
+
+    it("should render footnotes", () => {
+        renderPassage({footnotes: "Hello World"});
+
+        expect(screen.getByText("Hello World")).toBeInTheDocument();
+    });
+
+    it("should render first question instructions", () => {
+        renderPassage({passageText: "[[test]] Hello World"});
+
+        expect(screen.getByText("The symbol")).toBeInTheDocument();
+        expect(screen.getAllByText("[Marker for question test]")).toHaveLength(
+            2,
+        );
+        expect(
+            screen.getByText(
+                "indicates that question test references this portion of the passage",
+            ),
+        ).toBeInTheDocument();
+    });
```

Comment:
Question (non-blocking): I tried to figure out what's going on here, but I can't quite follow. How is "The symbol" appearing in the document? Same question for the other getByText statements here. Do these refer to something else?

Reply:
They're rendering the instructions for the widget; it's text generated by the component itself: https://github.com/Khan/perseus/blob/main/packages/perseus/src/widgets/passage.jsx#L415